tests/test_srttransform3d.py

```diff
@@ -36,3 +36,18 @@ def testMatrix():
     assert_array_almost_equal(tr3.getRotation()[1], tr.getRotation()[1])
     assert_array_almost_equal(tr3.getScale(), tr.getScale())
     assert_array_almost_equal(tr3.getTranslation(), tr.getTranslation())
+
+
+def testMapTypes():
```

Comment:
this would probably be much better as a parametrized test, I can make that change if you'd like.